
const signup = async ()=>{
 try{

 } catch(e: any){
    throw new Error(e);
 }
}

const signIn = async ()=>{
 try{

 } catch(e: any){
    throw new Error(e);
 }
}
export const AuthService = {
  signup,
  signIn
} 